manager/container.go

```diff
@@ -306,8 +322,8 @@ func newContainerData(containerName string, memoryCache *memory.InMemoryCache, h
 	cont := &containerData{
 		handler:                  handler,
 		memoryCache:              memoryCache,
-		housekeepingInterval:     *HousekeepingInterval,
-		maxHousekeepingInterval:  maxHousekeepingInterval,
+		housekeepingInterval:     jitter(*HousekeepingInterval, JitterFactor),
+		maxHousekeepingInterval:  jitter(maxHousekeepingInterval, JitterFactor),
```

Comment:
We should cap the housekeeping at `maxHousekeepingInternal`. I'd recommend not changing this.